CMakeLists.txt

```diff
@@ -33,6 +33,31 @@ TRIBITS_PACKAGE_DECL(Kokkos) # ENABLE_SHADOWING_WARNINGS)
 # HAVE_KOKKOS_DEBUG.  We define KOKKOS_HAVE_DEBUG here instead,
 # for compatibility with Kokkos' Makefile build system.
 
+if (TPL_ENABLE_CUDA)
+  if (DEFINED CUDA_VERSION)
+    if (CUDA_VERSION VERSION_GREATER_EQUAL "7.5")
+      set(KOKKOS_HAVE_CUDA_GEQ_75 TRUE)
+    endif()
+    if (CUDA_VERSION VERSION_GREATER_EQUAL "8.0")
+      set(KOKKOS_HAVE_CUDA_GEQ_80 TRUE)
+    endif()
+  endif()
+endif()
+
+set(Kokkos_ENABLE_Cuda_Lambda_DEFAULT OFF)
+if (KOKKOS_HAVE_CUDA_GEQ_75)
```

Comment:
How does this pick up the `KOKKOS_HAVE_CUDA_GEQ_80` ?

Reply:
see lines 36-45 above

Reply:
ah yes, `VERSION_GREATER_EQUAL` not equal